icepyx/core/query.py

```diff
@@ -19,8 +19,8 @@
 # QUESTION: why doesn't from granules import Granules as Granules work, since granules=icepyx.core.granules?
 # from icepyx.core.granules import Granules
 from icepyx.core.variables import Variables as Variables
-import icepyx.core.geospatial as geospatial
 import icepyx.core.validate_inputs as val
+import icepyx.core.spatial as sp
```

Comment:
Though technically you should not use `import scipy` over `from scipy import specific_function` ([e.g.](https://stackoverflow.com/questions/36014733/official-abbreviation-for-import-scipy-as-sp-sc)), if someone DOES `sp` is the most likely abbreviation they'll use. So I'd like to switch ours to `spat` or something to avoid any potential conflicts...